definitions/divorce/json/CaseEventToFields/CaseEventToFields-general-referral-nonprod.json

```diff
@@ -108,5 +108,137 @@
     "PageDisplayOrder": 1,
     "PageColumnNumber": 1,
     "ShowSummaryChangeOption": "Yes"
+  },
+  {
+    "LiveFrom": "20/10/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "generalReferralPayment",
+    "CaseFieldID": "GeneralReferralPaymentType",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "MANDATORY",
+    "PageID": "GeneralReferralPayment1",
+    "PageLabel": "General referral payment",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "ShowSummaryChangeOption": "No"
+  },
+  {
+    "LiveFrom": "20/10/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "generalReferralPayment",
+    "CaseFieldID": "SolicitorFeeAccountNumber",
+    "PageFieldDisplayOrder": 2,
+    "DisplayContext": "MANDATORY",
+    "PageID": "GeneralReferralPayment1",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "FieldShowCondition": "GeneralReferralPaymentType=\"feeAccount\""
+  },
+  {
+    "LiveFrom": "20/10/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "generalReferralPayment",
+    "CaseFieldID": "FeeAccountReference",
+    "PageFieldDisplayOrder": 3,
+    "DisplayContext": "OPTIONAL",
+    "PageID": "GeneralReferralPayment1",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "FieldShowCondition": "GeneralReferralPaymentType=\"feeAccount\""
+  },
+  {
+    "LiveFrom": "20/10/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "generalReferralPayment",
+    "CaseFieldID": "D8HelpWithFeesReferenceNumber",
+    "PageFieldDisplayOrder": 2,
+    "DisplayContext": "MANDATORY",
+    "PageID": "GeneralReferralPayment1",
+    "PageDisplayOrder": 2,
+    "PageColumnNumber": 1,
+    "FieldShowCondition": "GeneralReferralPaymentType=\"helpWithFees\""
+  },
+  {
+    "LiveFrom": "20/10/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "generalReferralPayment",
+    "CaseFieldID": "generalReferralWithoutNoticeFeeSummary",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "MANDATORY",
+    "PageID": "GeneralReferralPayment2",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1
+  },
+  {
+    "LiveFrom": "20/10/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "generalReferralPayment",
+    "CaseFieldID": "PaymentFeeAccountLabel",
+    "PageFieldDisplayOrder": 2,
+    "DisplayContext": "READONLY",
+    "PageID": "GeneralReferralPayment2",
```

Comment:
All the fields with "PageID: GeneralReferralPayment2"
should have "PageDisplayOrder: 2"
I think it's still working fine because the id is the same but technically they are all on page 2 as "GeneralReferralPayment2" is the second page :)

Reply:
Ahh okay! Ill double check them all now